### DISCUSSION TITLE
How to debug variables passed to sketchybar

### DISCUSSION THREAD

**Discussion by JaiDoubleU** [FelixKratz/SketchyBar#338] [Q&A] [answered]
Is there way of spitting out the values being passed to sketchybar?  I'm looking for something like console.log(var); 

Any help would me much appreciated

**Comment by FelixKratz** [comment, marked as answer]
Could you provide an example in which case exactly you would like to print variables passed to sketchybar? In regular scripts you can simply call `echo $VAR` and start sketchybar from commandline to see the output from the script.
